lib/enumerize/attribute.rb

```diff
@@ -17,6 +17,13 @@ def initialize(klass, name, options={})
         @default_value = find_default_value(options[:default])
         raise ArgumentError, 'invalid default value' unless @default_value
       end
+
+      # Define methods to get each value
+      @values.each do |v|
+        metaclass = class << self; self; end
+        metaclass.send(:define_method, v.to_s) { v.value }
+        metaclass.send(:define_method, v.to_s + '_text' ) { v.text }
+      end
```

Comment:
I think it's better to implement this methods using `method_missing` and `respond_to_missing?` as defining methods on the singleton class prevents objects being serialized using `Marshal.dump`. It might be an issue for some users of the gem.